src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTapeCorner.java

```diff
@@ -171,9 +171,17 @@ private void initBlock()
         setHardness(BLOCK_HARDNESS);
         setResistance(RESISTANCE);
         setLightOpacity(LIGHT_OPACITY);
-        setBlockUnbreakable();
     }
 
+    public boolean isReplaceable(IBlockAccess worldIn, BlockPos pos)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1161)